packages/zipkin-instrumentation-request-promise/src/request.js

```diff
@@ -92,13 +91,13 @@ const Request = class {
      * breaks zipkin scope
      */
     let cb = callback;
-    if (_.isFunction(callback)) {
+    if (typeof callback === 'function') {
```

Comment:
all of our other code uses this form. If we need a utility more robust, I think we should vendor the 4 lines